msteams-platform/samples/code-samples.md

```diff
@@ -8,8 +8,11 @@ keywords: Microsoft teams developer samples
 
 These code samples show you the capabilities of Microsoft Teams apps and various ways to implement those capabilities:
 
+<<<<<<< HEAD
```

Comment:
These are conflict markers. These are supposed to be removed while resolving the conflicts.